README.rst

```diff
@@ -0,0 +1,34 @@
+# PyLmod
+========
+:PyLmod: Python implementation of MIT Learning Modules API
+:Version: 0.0.1
+:Author: MIT Office of Digital Learning
+:Homepage: http://odl.mit.edu
+:License: BSD
+
+.. _Documentation: http://pylmod.readthedocs.org/en/latest/
+
+Dependencies:
+=============
+* Python 2.7+
+
+Development dependencies:
```

Comment:
I just wouldn't bother having this list at all. The setup.py and requirements.txt should declare the deps